frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -36,6 +37,12 @@ const HostsTableHeader = () => [
     transforms: [sortable],
     props: { className: tableColumnClasses[0] },
   },
+  {
+    title: 'Status',
+    // sortField: 'spec.machineRef.name',
+    // transforms: [sortable],
+    props: { className: tableColumnClasses[1] },
```

Comment:
Would be nice if this is sortable, but I don't think there's a way to add `sortFunc` from a plugin yet.